### ISSUE TITLE
Not working with kotlin after bumping gradle wrapper to 2.14-all and build tools to 2.2

### ISSUE THREAD

**Issue by rainqer** [arturdm/jacoco-android-gradle-plugin#26]
After bumping gradle wrapper to 2.14-all and build tools to 2.2, our test coverage went down from 58% to 9% overall. The tools successful runs all tests, then marks most of the packages as having 0% coverage.

**Comment by rainqer** [comment]
We are trying to tackle this problem

**Comment by julioromano** [comment]
I see the same behavior with build tools 2.2 and gradle 2.14 but even when using pure java (i.e. no kotlin). When using Android studio's code coverage feature I get a positive coverage, but when running `./gradlew jacocoTestReport` i get 0% in the produced reports.

**Comment by BrantApps** [comment]
Been seeing this for a while myself. Will keep you posted if I find anything out. I set-up the same system for our app at work and it isn't suffering the 0% coverage issue remarked here. Do our projects share some common structure? Might try and answer that tomorrow on the train to work and see if I can get the most simple of projects to work and then 'complicate' it with flavours, build types and multi-module make-up.

**Comment by BrantApps** [comment]
ok, so didn't make myself clear. I'm rolling my own Jacoco coverage report based off a blog post on the subject a year back. But bizarrely I have just got this working again by adding;

```
android {
    testOptions {
        unitTests.all {
            jacoco {
                includeNoLocationClasses = true
            }
        }
    }
}
```
...to my ````coverage.gradle```` file I'm used to add in the Jacoco task. This file now looks like [this](http://pastebin.com/Ufs9UkPW). 

Any help to getting your plugin working again?

**Comment by arturdm** [comment]
Fixed in [0.1.4](https://github.com/arturdm/jacoco-android-gradle-plugin/releases/tag/jacoco-android-release-0.1.4) release.
